doc/mgr/orchestrator.rst

```diff
@@ -115,6 +115,34 @@ Service Commands::
 
     ceph orch <start|stop|restart|redeploy|reconfig> <service_name>
 
+   .. note:: these commands applies to cephadm containerized daemons only.
+
+Options
+=======
+
+.. option:: start
+
+   Start the daemon on the corresponding host.
+
+.. option:: stop
+
+   Stop the daemon on the corresponding host.
+
+.. option:: restart
+
+   Restart the daemon on the corresponding host.
+
+.. option:: redeploy
+
+   Redeploy the ceph daemon on the corresponding host. This will recreate the daemon directory
```

Comment:
Drat! Ceph should be capitalized.  Since the parent has the same, I'll put in a followup PR to fix it in `main`